web/template/certificateprovider/enter_reference_number_opt_out.gohtml

```diff
@@ -0,0 +1,23 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "enterYourReferenceNumber" }}{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">{{ tr .App "enterYourReferenceNumber" }}</h1>
+
+            {{ trHtml .App "certificateProviderReferenceNumberOptOutContent" }}
+
+            <form novalidate method="post">
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset">
```

Comment:
again, can remove these. The `input` template does a form-group, and this isn't a fieldset